server/services/nl.py

```diff
@@ -160,13 +161,28 @@ def _temporal_classification(self, prediction) -> Union[NLClassifier, None]:
     return NLClassifier(type=ClassificationType.TEMPORAL, attributes=attributes)
 
   def _containedin_classification(self,
-                                  prediction) -> Union[NLClassifier, None]:
+                                  prediction, query: str) -> Union[NLClassifier, None]:
     if prediction != "Contained In":
       return None
 
+    contained_in_place_type = ContainedInPlaceType.PLACE
+    place_type_to_enum = OrderedDict({
+      "county": ContainedInPlaceType.COUNTY,
```

Comment:
@jehangiramjad  let me know if the orders of detection are correct (ex: county before city)

Reply:
Good question. Given that we have a lot more data about counties vs cities, having that as top is good. How about:

1. County
2. State
3. Country
4. City

The rest are fine. as they are. 

But also, please also look for plurals or some sort of regex so that plurals can also be matched?